### DISCUSSION TITLE
Add BigQuery to Postgres transfer

### DISCUSSION THREAD

**Discussion by JoeSham** [apache/airflow#27817] [Ideas]
**Description**
Hi, I suggest to add a BigQuery to Postgres transfer. There are currently 2 similar transfers already available: BigQuery to MySQL and BigQuery to MSSQL.
https://github.com/apache/airflow/tree/main/airflow/providers/google/cloud/transfers

**Use case/motivation**
To transfer data from BigQuery to Postgres

**Related issues**
Have not found any

**Are you willing to submit a PR?**
- [x] Yes I am willing to submit a PR!

**Code of Conduct**
- [x] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
Feel free to implement those.
